overlord/snapstate/backend/snapdata.go

```diff
@@ -33,8 +33,8 @@ import (
 )
 
 // RemoveSnapData removes the data for the given version of the given snap.
-func (b Backend) RemoveSnapData(snap *snap.Info) error {
-	dirs, err := snapDataDirs(snap)
+func (b Backend) RemoveSnapData(snap *snap.Info, opts *dirs.SnapDirOptions) error {
```

Comment:
Shouldn't we add a member with `dirs.SnapDirOptions` type into the `snap.Info` type itself? At least at first sight, it would appear logical to me if all informations pertaining a snap's installation were accessible via the `snap.Info` type.

Reply:
Ah, nevermind, I think I got it: the information on the installation is in the `state.State` type, whereas `snap.Info` is only about static information.